includes/modules/payment/paypal/paypalwpp_incontext.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * paypal EC In-Context button display template
+ *
+ * @package paymentMethod
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: Author: mc12345678  Wed Aug 17 03:03:03 2016 -0500 Modified in v1.6.0 $
+ */
+
+    require_once(DIR_WS_CLASSES . 'payment.php');
+    $paypalwpp_module = 'paypalwpp';
+    // init the payment object
+    $classPay = new payment($paypalwpp_module);
+?><script>
+window.paypalCheckoutReady = function () {
+  paypal.checkout.setup('<?php echo MODULE_PAYMENT_PAYPALWPP_MERCHANTID; ?>', {
+    //locale: '<?php echo ${$paypalwpp_module}->getLanguageCode('incontext'); ?>',
```

Comment:
My bad. I had the global mixed around in my head. (Not happy with all the global vars we use in ZC, but we're working on changing that. Separate topic.)
You're right. No need to switch to ${$classPay}.

If we decide later to clean up the string to avoid assigning another unused var, we can do that in the snippet you quoted as well, at that time.